red/part2/part2_learner/mtl_red_part_2_see.md

```diff
@@ -57,9 +57,9 @@ output:
      - In the event they do not, to make clinic selections yourself, you can select the same set of clinics you chose in Part 1 of _Modeling to Learn_.
```

Comment:
Hi @mnallajerla - I couldn't make a comment in some of the lines , so I'm making it all in line 57 for now:
- [ ] Line 54: Change "At mtl.how/data_test..." to "At mtl.how/data..."
- [ ] Line 44: Language is implying that we're making Clinic Selections again, would rephrase it "
- [ ] General: When not referring to Clinic Selection as the tab, I would include the space (e.g. How we write Clinic Selections in Line 42 and 44 is different than line 56. But in Line 55, I would keep the space since it's referring to the ClinicSelection tab).
- [ ] Continue working on editing 2.0 or 2.5 to 3.0

Lmk if you have any questions, thanks!
_________________________________________________________________________
For @lijenn to do BEFORE merging pull request:
- [ ] add data ui cheatsheet to red part ii and blue s3 folders
- [ ] add cpt code cheatsheet 
- [ ] facilitator one pager